openstack/identity/v3/projects/doc.go

```diff
@@ -64,5 +64,32 @@ Example to Delete a Project
 	if err != nil {
 		panic(err)
 	}
+
+Example to Check if a Project contains a Tag
+
+	projectID := "966b3c7d36a24facaf20b7e458bf2192"
+	tag:= "atag"
+	err := projects.GetProjectTag(context.TODO(), identityClient, projectID,tag).ExtractErr()
```

Comment:
Please leave a space after the comma.
```suggestion
	err := projects.GetProjectTag(context.TODO(), identityClient, projectID, tag).ExtractErr()
```
Also in other docs below.